test/unit/util/operator/test-operators-identity-project.cpp

```diff
@@ -0,0 +1,83 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// Source file containing unit tests for Operators.
+///
+
+#include "RAJA_test-base.hpp"
+#include "RAJA_unit-test-types.hpp"
+
+template<typename T>
+class OperatorsUnitTestIdentityProject : public ::testing::Test {};
+
+TYPED_TEST_SUITE(OperatorsUnitTestIdentityProject, UnitIntFloatTypes);
+
+template<typename T>
+void identity_test()
+{
+  using Ident = RAJA::operators::identity<T>;
+
+  Ident id;
+  T i = static_cast<T>(0);
+  T j = static_cast<T>(1);
+  ASSERT_EQ(id(i), T(0));
+  ASSERT_EQ(id(j), T(1));
+
+  if (std::is_signed<T>::value) {
+    j = static_cast<T>(-1);
+    ASSERT_EQ(id(j), T(-1));
+  }
+}
+
+template<typename T>
+void project1st_test()
+{
+  using Proj1 = RAJA::operators::project1st<T, T>;
```

Comment:
@rhornung67 The file and test names refer to this operator type and the one used on line 58